js/lite/README.md

```diff
@@ -0,0 +1,196 @@
+# Gradio-Lite: Serverless Gradio Running Entirely in Your Browser
+
+Tags: SERVERLESS, BROWSER, PYODIDE
+
+Gradio is a popular Python library for creating interactive machine learning apps. Traditionally, Gradio applications have relied on server-side infrastructure to run, which can be a hurdle for developers who need to host their applications. 
+
+Enter Gradio-lite (`@gradio/lite`): a library that leverages [Pyodide](https://pyodide.org/en/stable/) to bring Gradio directly to your browser. In this blog post, we'll explore what `@gradio/lite` is, go over example code, and discuss the benefits it offers for running Gradio applications.
+
+## What is `@gradio/lite`?
+
+`@gradio/lite` is a JavaScript library that enables you to run Gradio applications directly within your web browser. It achieves this by utilizing Pyodide, a Python runtime for WebAssembly, which allows Python code to be executed in the browser environment. With `@gradio/lite`, you can **write regular Python code for your Gradio applications**, and they will **run seamlessly in the browser** without the need for server-side infrastructure.
+
+## Getting Started
+
+Let's build a "Hello World" Gradio app in `@gradio/lite`
```

Comment:
If this is meant for https://www.npmjs.com/package/@gradio/lite, then we should mention you can install Gradio-Lite in two ways, either using `npm i` or using the CDN, and say that the examples below are using the CDN but you can also use with `npm`

Reply:
@abidlabs the thing is i'm not sure it works with npm yet, like @pngwn was saying. Maybe @whitphx can comment but i haven't ever used it outside of a cdn

Reply:
Sg lets just mention that the best way to use `@gradio/lite` right now is through the CDN